internal/sourcecode/templates/laravel/standard/docker/nginx.conf

```diff
@@ -0,0 +1,108 @@
+user nginx;
```

Comment:
Perhaps to avoid duplicating this config across laravel runtimes, we could have a separate path in `templates/laravel` for common files, which could also be nginx includes if these `nginx.conf` files are not identical.